vyper/semantics/analysis/utils.py

```diff
@@ -309,7 +309,11 @@ def types_from_List(self, node):
             # subtype can be anything
             types_list = types.PRIMITIVE_TYPES
             # 1 is minimum possible length for dynarray, assignable to anything
-            ret = [DArrayT(t, 1) for t in types_list.values()]
+            # for type classes like bytestrings, use a generic type acceptor
```

Comment:
i think for hygiene we want a little bit more safety, like checking in the else branch if it's actually a subclass of VyperType

Reply:
i pushed this hygiene change in https://github.com/vyperlang/vyper/pull/3377/commits/0fc474446154fb871eb071b0df6be30ca1c0f725